src/main/java/com/github/charithe/kafka/KafkaJunitRule.java

```diff
@@ -129,6 +140,28 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
         }
     }
 
+    /**
+     *
+     * Shutdown Kafka Broker before the test termination to test consumer exceptions
+     *
+     */
+    public void shutdownKafka() {
+        if (kafkaServer != null) {
+            LOGGER.info("Shutting down Kafka Server");
+            kafkaServer.shutdown();
+        }
+    }
+
+    /**
+     * Restarts the server
+     */
+    public void start(){
```

Comment:
The shutdown method is called `shutdownKafka` so this should then be called `startKafka`